src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -42,26 +41,35 @@ typedef struct uct_ib_mlx5_dbrec_page {
 
 static ucs_status_t uct_ib_mlx5_reg_key(uct_ib_md_t *md, void *address,
                                         size_t length, uint64_t access_flags,
-                                        uct_ib_mem_t *ib_memh)
+                                        uct_ib_mem_t *ib_memh, int mr_index)
 {
     uct_ib_mlx5_mem_t *memh = ucs_derived_of(ib_memh, uct_ib_mlx5_mem_t);
-    ucs_status_t status;
 
-    status = uct_ib_reg_mr(md->pd, address, length, access_flags, &memh->mr);
-    if (status != UCS_OK) {
-        return status;
-    }
+    return uct_ib_verbs_reg_key_impl(md, address, length, access_flags, ib_memh,
+                                     &memh->mrs[mr_index].super, mr_index);
```

Comment:
how come mlx5 code is using "verbs" registration?

Reply:
to register mtt mr mlx5 code using verbs reg, common code.

Reply:
as noted in other place - need to rename this helper function